### DISCUSSION TITLE
Differentiate between 'no info' and 'feature: false'

### DISCUSSION THREAD

**Discussion by luroc** [adamghill/unsuckjs.com#10] [Ideas]
I was a bit irritated, that all those frameworks would announce that they are not compatible with IE 11, checked on the first one (strawberry) and couldn't find a source. Guessing by the meta data, the green tick means "there is info that IE 11 is supported" and the cross means "there is no info, that IE11 is supported".   
The same goes for the web components row, but here there even is one framework with the data of "web_components": false, but it's not reflected in the UI

I'd propose to leave out the 'X' if there is no info and only to set the 'X' if the attribute is 'false'.

**Comment by adamghill** [comment]
Thanks for making this and I think you are right. I'll make this more clear soon!

**Comment by adamghill** [comment]
Fixed in 4a1a8e6a and deployed live. Thanks again for the suggestion.

**Comment by luroc** [comment]
Nice, thanks for the work!
